apps/condo/domains/news/components/RecipientCounter.tsx

```diff
@@ -61,6 +62,10 @@ const isTargetedToUnitName = ({ property, unitType, unitName }: TNewsItemScopeNo
     !!property && !!unitType && !!unitName
 )
 
+const isAllOrganization = (newsItemScopes: TNewsItemScopeNoInstance[]) => {
+    return filter(newsItemScopes, { property: null }).length > 0
```

Comment:
Technically, the unitType may be not null (news item for all flats or parkings). In this case, you erroneously detect the all-organization news item.
```suggestion
    return filter(newsItemScopes, { property: null, unitType: null, unitName: null }).length > 0
```

Reply:
done